src/content/get-started/fundamentals/dart.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Intro to Dart
+description: Learn about the Dart programming language
+prev:
+  title: Fundamentals
+  path: /get-started/fundamentals
+next:
+  title: Widgets
+  path: /get-started/fundamentals/widgets
+---
+
+To get started with Flutter, 
+you need to have some familiarity with 
+the Dart programming language, which Flutter 
+applications are written in.
+This page is a gentle introduction to Dart, 
+and if you're comfortable reading the 
+code examples, feel free to skip this page. 
+You do not need to be an expert in Dart to 
+continue with this series.
+
+## Dart
+
+Flutter applications are built in [Dart][],
+a language that will look familiar
+to anyone who's written Java, Javascript,
+or any other C-like language.  
+
+:::note
+Installing Flutter also installs Dart,
+so you don't need to install Dart separately.
+:::
+
+The following example is a small program that 
+fetches data from dart.dev, 
+decodes the returned json, 
+and prints it to the console. 
+If you’re confident in your ability to 
+understand this program, 
+feel free to skip to the page.
+
+```dart
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+class Package {
+  final String name, latestVersion; 
+  String? description;
```

Comment:
I believe many developers end up including lints to discourage multiple declarations in one line, so consider avoiding it here:

```suggestion
  final String name;
  final String latestVersion; 
  final String? description;
```